libs/shared/storage/src/lib/simulation-storage.service.ts

```diff
@@ -36,7 +37,17 @@ export class SimulationStorageService {
     );
     const output = await this.storage.deleteObject(fileObject);
 
-    // TODO check deletion worked
+    await this.storage.getObject(fileObject)
+      .then((file: AWS.S3.GetObjectOutput): void => {
+        if (!file.DeleteMarker) {
```

Comment:
@jonrkarr  Were you able to get this to work? 
As far as I could tell, this approach does not work and will always throw an error. The delete marker does not get set on files when deleted this way. I believe it has something to do with the way versioning is used on the s3 bucket

Reply:
The combination seemed to work

Reply:
Was this on the UConn bucket? As far as I can tell from the docs, the delete marker is only set for versioned objects. If the bucket is not versioned, the file should not exist. If the tag is set, then maybe versioning is enabled on the bucket?